tyk-control-plane/values.yaml

```diff
@@ -1337,6 +1337,8 @@ tyk-dev-portal:
   securityContext:
     runAsUser: 1000
     fsGroup: 2000
+    runAsNonRoot: true
```

Comment:
You have added it for Dev Portal pod only. Don't we need to add it for other components?
Also, you haven't added it in Dev Portal component chart